frontend/src/scenes/data-warehouse/new/dataWarehouseTableLogic.tsx

```diff
@@ -45,6 +45,16 @@ export const dataWarehouseTableLogic = kea<dataWarehouseTableLogicType>([
         setDataTableQuery: (query: DataTableNode) => ({ query }),
     }),
     loaders(({ props }) => ({
+        tables: [
+            [] as DataWarehouseTable[],
+            {
+                loadTables: async () => {
+                    return await api.dataWarehouseTables
+                        .list({ exclusion_url_pattern_prefix: 'https://posthog-s3-datawarehouse' })
```

Comment:
Hmm, I would shy away from this - we're just asking for trouble trying to do some string matching here. A `table` model without a corresponding `schema` model is how we identify these tables - is there a reason we're not running a migration to add some status/modifier/type to the `table` table to differentiate these?

Reply:
Ah, ok, I'll do it with a migration.

Reply:
For the self managed sources we do not seem to have other table linkings as far as I can tell, I did see that there is a `created_by_id` column that is null in the case that our table was not created by a user.